src/Web/Sfa.Das.Sas.Shared.Components.Web/ViewComponents/Fat/Search/SearchViewComponent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Sfa.Das.Sas.Shared.Components.ViewComponents
+{
+    public class FatSearchViewComponent : ViewComponent
```

Comment:
I assume these will move into the shared component library

Reply:
yep, this is a simple test one for now while the projects get set up correctly